web-services/client/src/main/java/datawave/webservice/query/map/QueryGeometry.java

```diff
@@ -0,0 +1,68 @@
+package datawave.webservice.query.map;
+
+import org.apache.commons.lang.StringEscapeUtils;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.io.Serializable;
+
+/**
+ * This represents a datawave geo function and the associated query geometry in GeoJSON format.
+ */
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.NONE)
+public class QueryGeometry implements Serializable {
+    
+    @XmlElement
+    private String function;
+    @XmlElement
+    private String geometry;
+    
+    public QueryGeometry(String function, String geometry) {
+        this.function = function;
+        this.geometry = geometry;
+    }
+    
+    public String getFunction() {
+        return function;
+    }
+    
+    public void setFunction(String function) {
+        this.function = function;
+    }
+    
+    public String getGeometry() {
+        return geometry;
+    }
+    
+    public void setGeometry(String geometry) {
+        this.geometry = geometry;
+    }
+    
+    String toGeoJsonFeature() {
```

Comment:
Should this be `protected`?

Reply:
What is going on with gitlab?  I added a comment on here saying that it can be public, but it isn't showing up...